toolkit/Makefile

```diff
@@ -64,7 +64,7 @@ PROJECT_ROOT     ?= $(realpath $(toolkit_root)/..)
 BUILD_DIR        ?= $(PROJECT_ROOT)/build
 OUT_DIR          ?= $(PROJECT_ROOT)/out
 SPECS_DIR        ?= $(PROJECT_ROOT)/SPECS
-CCACHE_DIR       ?= $(PROJECT_ROOT)/ccache
+CCACHE_DIR       ?= /home/$(SUDO_USER)/mariner-ccache
```

Comment:
Approved to unblock. However, would love to see some sort of check that `$(SUDO_USER)` is not empty and that `/home/$(SUDO_USER)` already exists before we create the folder.